commands/agent-upgrade.py

```diff
@@ -94,14 +96,29 @@ def save_rollback_info():
         shutil.copy(agent_conf, backup_path)
 
 def find_new_tools():
-    dirs = [name for name in os.listdir(UPGRADE_DIR) if path.isdir(path.join(UPGRADE_DIR, name))]
+    dirs = [name for name in os.listdir(UPGRADE_DIR) if path.join(UPGRADE_DIR, name).endswith('.tgz')]
```

Comment:
s/dirs/files/ ?

Reply:
This is a holdover from the tools originally being pushed out unpacked and moved into place as directories - I'll change it.